dask-gateway-dev.yaml

```
@@ -0,0 +1,58 @@
+# OPTIONAL conda environment file to help get setup with
+# dependencies to build + test dask-gateway locally
+name: dask-gateway-dev
+channels:
+  - conda-forge
+dependencies:
+  # python and pip
+  - python=3.9
+  - pip  
+  
+  # Dependencies to build dask-gateway and dask-gateway-server
+  # NOTE: changes to this block must also be reflected in the following files 
+  # - dask-gateway/setup.py
+  # - dask-gateway-server/setup.py
+  - aiohttp
+  - dask
+  - distributed
+  - cryptography
+  - tornado
+  - traitlets
+  - colorlog
+  - sqlalchemy # see: dask-gateway-server/setup.py extras_require
+  - pykerberos # see: dask-gateway-server/setup.py extras_require 
+  - go=1.17.8  # golang version to  build dask-gateway-server/dask-gateway-proxy
+  - go-cgo     # golang associated dependency to assist with local setup
+  - compilers  # golang associated dependency to assist with local setup
+  
+  # Dependencies required to build
+  # the dask-gateway documentation
+  - skein
+  - sphinx
+  - dask-sphinx-theme
+  
+  # Dependencies used in CI and running tests locally:
+  # dask-gateway/continuous_integration/kubernetes/install.sh
+  - black
+  - colorlog
+  - ipywidgets
+  - jupyterhub
+  - notebook
+  - trustme
+  - flake8
+  - pytest
+  - pytest-asyncio
```

`pre-commit` is a tool used instead now, so `black` and `flake8` was removed for example.

`colorlog` is part of install_requires for dask-gateway-server and is added above instead.

```suggestion
  - ipywidgets
  - jupyterhub
  - notebook
  - trustme
  - pytest
  - pytest-asyncio
```